.babelrc

```diff
@@ -1,7 +1,13 @@
 {
   // Javascript settings.
   "presets": [
-    "es2015",
+    ["env", {
+      "targets": {
+        "uglify": true,
+        "browsers": ["last 2 versions", "ie 11", "not ie 10"]
```

Comment:
Are the last two versions our officially supported browsers? (Do we keep that information anywhere?)